include/swift/AST/ActorIsolation.h

```diff
@@ -140,7 +135,7 @@ class ActorIsolation {
             .Case("global_actor",
                   std::optional<ActorIsolation>(ActorIsolation::GlobalActor))
             .Case("global_actor_unsafe", std::optional<ActorIsolation>(
-                                             ActorIsolation::GlobalActorUnsafe))
+                                             ActorIsolation::GlobalActor))
```

Comment:
Should this mark ActorIsolation as preconcurrency and "isSILParsed" set to `true`?

Reply:
This line of code only sets the actor isolation kind; the resulting `ActorIsolation` is marked `true` for "is SIL parsed" in the return statement below. If `@preconcurrency` is not preserved for `GlobalActor`, I don't think we should preserve it for `GlobalActorUnsafe`.

I also think that this `global_actor_unsafe` handling can just be removed; it doesn't appear to be used anywhere in SIL tests.

Reply:
If it doesn’t appear in the tests it might be best to just remove it then…